server/server.ts

```diff
@@ -0,0 +1,177 @@
+import * as express from 'express';
```

Comment:
I rewrote the server in TypeScript using promises and tried to simplify things a bit. I did as direct a rewrite as possible from what you had before.